interop/tapir/src/main/scala/caliban/interop/tapir/ws/Protocol.scala

```diff
@@ -275,7 +274,9 @@ object Protocol {
                                    subscriptions.untrack(id)
                                  case GraphQLWSInput(Ops.ConnectionTerminate, _, _)   =>
                                    ZIO.interrupt
-                               }.runDrain.interruptible
+                                 case _                                               =>
+                                   ZIO.unit
+                               }.interruptible
                                  .catchAll(_ => output.offer(Right(connectionError)))
```

Comment:
Technically we could use `orElse` here as well, which might be slightly more memory efficient

Reply:
Updated it 😄